c/ldv-memsafety/memleaks_test23_1_true-valid-memsafety.i

```diff
@@ -1062,8 +1062,8 @@ int ldv_nonpositive(void) {
 
 int ldv_positive(void) {
  int r = __VERIFIER_nondet_int();
- if(r>0) return r;
- else return 1;
+ if(r>255) return r;
+ else return 256;
```

Comment:
Could we put this condition to the caller place? Because the name of the function suggests that it returns >=1